Amethyst/Managers/WindowManager.swift

```diff
@@ -334,8 +334,14 @@ final class WindowManager<Application: ApplicationType>: NSObject, Codable {
         guard let screen = window.screen() else {
             return
         }
+        let windowIDsArray = [NSNumber(value: window.cgID() as UInt32)] as NSArray
```

Comment:
Could you make this a function on `CGWindowsInfo` or `CGSpacesInfo`? The manager is generic to aid in further testing, which requires isolating accesses to application specific functions like the CGS copy functions.

Reply:
> Could you make this a function on `CGWindowsInfo` or `CGSpacesInfo`? The manager is generic to aid in further testing, which requires isolating accesses to application specific functions like the CGS copy functions.

Of course, I'll do it later, I'm preparing my exam😭.

Reply:
No worries. Good luck with your exam!